app/experimenter/experiments/api/v5/serializers.py

```diff
@@ -191,43 +217,63 @@ class NimbusStatusRestrictionMixin:
                 if set(data.keys()) != {"status"}:
                     raise serializers.ValidationError(
                         {
                             "experiment": [
-                                f"Nimbus Experiment has status '{status}', only "
+                                f"Nimbus Experiment has status {status}, only "
                                 "status can be changed."
                             ]
                         }
                     )
-            elif status not in self.ALLOWS_UPDATE:
-                required_statuses = ", ".join(self.ALLOWS_UPDATE)
+            elif status not in self.ALLOWS_STATUS_UPDATE:
+                required_statuses = ", ".join(self.ALLOWS_STATUS_UPDATE)
                 raise serializers.ValidationError(
                     {
                         "experiment": [
-                            f"Nimbus Experiment has status '{status}', but can only "
-                            f"be changed when set to '{required_statuses}'."
+                            f"Nimbus Experiment has status {status}, but can only "
+                            f"be changed when set to {required_statuses}."
                         ]
                     }
                 )
 
         return data
 
-    def validate_status(self, value):
-        if (
-            self.instance
-            and value != self.instance.status
-            and value not in self.VALID_STATUS_TRANSITIONS.get(self.instance.status, ())
-        ):
-            status = self.instance.status
-            raise serializers.ValidationError(
-                f"Nimbus Experiment status cannot transition from {status} to {value}."
-            )
 
-        return value
+class PublishStatusValidationMixin(BaseStatusValidationMixin):
+    ALLOWS_EXPERIMENT_CHANGES = (NimbusExperiment.PublishStatus.IDLE,)
+    FIELD = "publish_status"
+    VALID_TRANSITIONS = {
+        NimbusExperiment.PublishStatus.IDLE: (NimbusExperiment.PublishStatus.REVIEW,),
+        NimbusExperiment.PublishStatus.REVIEW: (
+            NimbusExperiment.PublishStatus.IDLE,
+            NimbusExperiment.PublishStatus.APPROVED,
+        ),
+    }
+
+    def validate(self, data):
+        data = super().validate(data)
+
+        if self.instance:
+            publish_status = self.instance.publish_status
+            if publish_status not in self.ALLOWS_EXPERIMENT_CHANGES and set(
+                data.keys()
+            ) != {"publish_status"}:
+                required_publish_statuses = ", ".join(self.ALLOWS_EXPERIMENT_CHANGES)
+                raise serializers.ValidationError(
+                    {
+                        "experiment": [
+                            f"Nimbus Experiment has publish status {publish_status} but "
+                            f"can only be changed with {required_publish_statuses}."
+                        ]
+                    }
+                )
+
+        return data
```

Comment:
Isn't this case the same in both as well?

Reply:
I think this is rejecting general experiment updates outside of IDLE publishing status. 

The other serializer rejects general experiment updates outside of DRAFT experiment status.